### ISSUE TITLE
Listen on multiple routes

### ISSUE THREAD

**Issue by edi9999** [thoov/mock-socket#251]
Currently I think it is only possible to listen on one ws url.

For example : 

```
const ws = new WebServer('/path')
```

I personally want to listen to multiple paths, but without changing the server itself.

This could be achieved with regexes, for example : 

```
const ws = new WebServer(/(\/path|\/otherpath)/)
```

**Comment by sechel** [comment]
Good idea: We have a scenario where we connect to the same server with different sets of query parameters. Cannot do this atm with mock-socket.
